src/Grid/grid.jl

```diff
@@ -765,8 +765,11 @@ for INDEX in (:VertexIndex, :EdgeIndex, :FaceIndex)
         #To be able to do a,b = faceidx
         Base.iterate(I::($INDEX), state::Int=1) = (state==3) ?  nothing : (I[state], state+1)
 
-        #For (cellid, faceidx) in faceset
-        Base.in(v::Tuple{Int, Int}, s::Set{$INDEX}) = in($INDEX(v), s)
+        #necessary for (cellid, faceidx) in faceset
```

Comment:
When reading this I first thought `for` referred to a for loop, but I think the unpacking is done elsewhere.
```suggestion
        #Necessary to check if, e.g. `(cellid, faceidx) in faceset`
```